src/record/record.js

```diff
@@ -449,7 +446,16 @@ Record.prototype._applyUpdate = function( message ) {
  */
 Record.prototype._onRead = function( message ) {
 	this.version = parseInt( message.data[ 1 ], 10 );
-	this._applyChange( jsonPath.set( this._$data, undefined, JSON.parse( message.data[ 2 ] ) ) );
+
+	this._$data = JSON.parse( message.data[ 2 ] );
+	if ( this._eventEmitter._callbacks ) {
+		var paths = Object.keys( this._eventEmitter._callbacks );
+
+		for ( var i = 0; i < paths.length; i++ ) {
+			this._eventEmitter.emit( paths[ i ], this.get( paths[ i ] ) );
```

Comment:
That was my point with my previous comment ( which I should have spent a bit more time on! ),

Before we always updated all initial paths with `triggerNow=true`, regardless if undefined or not. 
If `triggerNow=false` we never notified paths when undefined.

The new change either notifies everything with undefined if `triggerNow=true` and `isReady=false`, or with the previous comment, will never notify anyone if is `triggerNow=false`

Generally this was an inconsistency in the code previously, I'm just wondering if:
a) It creates any compatibility issues for those not using `isSubscribed`
b) If telling people it is undefined initially is desired, as some might take it as an indication to actually populate missing fields.

Will need to discuss with team to see what the general agreement is. Otherwise, this fixes the bug #229 well!